### ISSUE TITLE
Warning key combo already in use by other program is not accessible to screen readers

### ISSUE THREAD

**Issue by molaachir** [aleab/toastify#77]
Expected behaviour: At some point, when tabbing through the settings pane, I would like to be able to hear the warning that a specific chozen key combo is already in use by another application. Now, I have to try key combo byu key combo, to find out what works
